enterprise/reporting/utils.go

```diff
@@ -104,3 +131,25 @@ func getPIIColumnsToExclude() []string {
 	}
 	return piiColumnsToExclude
 }
+
+func sanitizeJSONForReports(input json.RawMessage) (json.RawMessage, error) {
```

Comment:
We appear to already have more than enough similar sanitization functions:

1. `jobsdb.sanitizeJSON`
2. `whutils.SanitizeJSON`

I would propose to extract this function to rudder-go-kit and use it wherever necessary

Reply:
Will be following up on this [here](https://linear.app/rudderstack/issue/PIPE-1834/gateway-validation-library)